source/client-side-operations-timeout/tests/command-execution.yml

```diff
@@ -27,7 +27,7 @@ initialData:
 tests:
   - description: "maxTimeMS value in the command is less than timeoutMS"
     operations:
-      # Artificially increase the server RTT to ~10ms.
+      # Artificially increase the server RTT to ~20ms.
```

Comment:
> calculating 90th percentile RTT requires at least 10 RTT samples

If there's less than 10 samples, wouldn't the 90th percentile RTT be approximated by the max() or the 2nd max()? Using 0 when there's less than 10 samples is exactly the kind of bug that this test aims to find. 

In Python I'm currently using the average RTT (the same value used during server selection to satisfy localThresholdMS).

> Even if we use average RTT as our measurement, I'd still like to get at least 5 samples to prevent noisy samples on startup from artificially increasing RTT.

I don't agree. The worst that happens with an "artificially increased RTT" is that the operation gets rejected unnecessarily but the connection can be reused. The worst that happens with using 0 or min(RTT) is that the operation fails with a socket timeout and the connection needs to be closed. The second one is much worse since connection establishment is expensive.

Reply:
> If there's less than 10 samples, wouldn't the 90th percentile RTT be approximated by the max() or the 2nd max()?

Discussing offline now, but I'd be in favor of changing Go to use average RTT for both `maxTimeMS` calculation and "short-circuiting". In that case, we wouldn't have a minimum sample requirement.